ligo/stablecoin/operator.ligo

```diff
@@ -13,7 +13,7 @@
 function validate_operator_owner_is_sender
   ( const param : operator_param
   ) : unit is if param.0 = Tezos.sender
-  then unit else failwith ("NOT_OWNER")
+  then unit else failwith ("NOT_TOKEN_OWNER")
```

Comment:
I have cleaned up some errors in #61. Apparently all contract changes in this PR only modify some error messages. If there is a good reason to update some message, let's do it, but also update `specification.md`. If there is no _really good_ reason, let's not change error messages because some contract version has been released already and I think we should make only really necessary changes, not cosmetic ones.

Reply:
As of now, after rebasing, this PR does not contain any contracts changes